Sources/SwiftKuery/QueryResult.swift

```diff
@@ -92,6 +105,8 @@ public enum QueryResult {
                 }
                 return dictionary
             }
+            // As this function consumes the entire result set we can explicitly close the result set allowing the release of the underlying connection.
+            resultSet.done()
```

Comment:
We need to update the documentation for `asRows` to indicate that it fully consumes the result set _and_ releases the underlying connection so that it can be reused (ie. the user does not have to call `resultSet.done()` explicitly).